applications/GeoMechanicsApplication/tests/test_truss_backbone_mat.py

```diff
@@ -0,0 +1,125 @@
+import os
+
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import test_helper
+
+
+class KratosGeoMechanicsTrussBackboneMaterialTests(KratosUnittest.TestCase):
+    """
+    This class contains tests for the truss backbone material
+    """
+
+    def setUp(self):
+        # Code here will be placed BEFORE every test in this TestCase.
+        pass
+
+    def tearDown(self):
+        # Code here will be placed AFTER every test in this TestCase.
+        pass
+
+    def test_truss_backbone_mat_tension(self):
+        """
+        2 element elongation test. The truss backbone material has a linear backbone with a
+        stiffness of 100.0 [N/m^2] and an un- and reloading stiffness of 200.0 [N/m^2].
+        The 2 element rod is elongated from 2 to 3 m, then compressed to 2.75 m, followed by
+        elongation to 4 m. The resulting force displacement curve should initially follow
+        the backbone, unload elastically, reload elastically, then follow the backbone further.
+        """
+        test_name    = r'truss_backbone_material\tension'
```

Comment:
I would suggest to change this to a generic path (rather than the Windows-style path it is now):
```suggestion
        test_name    = 'truss_backbone_material/tension'
```

Reply:
Replaced by using `os.path.join`.